### ISSUE TITLE
Pins definition

### ISSUE THREAD

**Issue by ssombra** [DavidMarzocca/NMLB#1]
Looking forward to build your project. After just having a look at the code, in the section where the Pins are defined, shouldn't be CHOut instead of CEOut?

**Comment by DavidMarzocca** [comment]
Hi, yes thanks a lot! I corrected this on my local version when I compiled the code but forgot to update it here. Now the variable is defined as CEout (Charge Enabler output).
The variable controls when to turn on or off the TP4056. I set it up so that the lipo gets charged at max ~4.0V, which should prolong its lifetime if it is left for long periods without use, at the price of a slightly smaller autonomy (it's extremely long in any case..).

**Comment by ssombra** [comment]
Good. And talking about the TP4056, do I need to change something in the code if I am not using it?

**Comment by DavidMarzocca** [comment]
No, you can leave it as it is. Simply in that case it won't matter if the CEout pin is enabled or disabled, you can also remove those lines if you want.

**Comment by ssombra** [comment]
Thanks. I'll give it a try.
